src/plugin.py

```diff
@@ -30,6 +30,28 @@
 
 
 class BNetPlugin(Plugin):
+    async def get_game_time(self, game_id, context):
+        gameTimeMinutes = 0
+        log.info('game_id: ' + game_id)
+        if game_id == "5272175": # Overwatch
+            log.debug("Fetching playtime for Overwatch...")
+            player_data = await self.backend_client.get_ow_player_data()
+            if player_data == None:
+                log.error('No Overwatch profile found.')
+                return GameTime(game_id, gameTimeMinutes, None)
+            if player_data['private'] == True:
+                log.info('Unable to get data as Overwatch profile is private.')
+                return GameTime(game_id, gameTimeMinutes, None)
+            qpTime = player_data['playtime']['quickplay']
+            if qpTime.count(':') == 1: # minutes and seconds
+                match = re.search('(?:(?P<m>\\d+):)(?P<s>\\d+)', qpTime)
```

Comment:
why not using time library for this task? Regex is less obvious for me (but if you'll write good tests I'm OK with this)

Reply:
I tried: strptime is limited to a range of 0..23. The hours in playtime is not. A playtime like 100:00:00 will throw an error when processing it with strptime and %H:%M:%S

Reply:
As for tests I'd like to add unit tests preferably, but I haven't found a good way of mocking the server reponse.

Reply:
you can simply add json response to `tests/website_mock`.